test_selinux.py

```diff
@@ -0,0 +1,26 @@
+"""
+This test is to check restorecon command for selinux context
+"""
+
+import unittest
+from .testutils import system
+
+
+class selinux_test(unittest.TestCase):
+    """
+    selinux_test class
+    """
+    def test_selinux(self):
+        """
+        actual test case
+        """
+        out, err, retcode = system("ls -lZ /etc/machine-id")
+        out = out.decode('utf-8')
+        out1, err1, retcode1 = system("restorecon -v /etc/machine-id")
```

Comment:
We are not using `out1, err1, retcode1` anywhere in this file.
I think we can just invoke the method `system("restorecon -v /etc/machine-id")` instead. 
i.e, Only `system("restorecon -v /etc/machine-id")` instead of `out1, err1, retcode1 = system("restorecon -v /etc/machine-id")`

@kushaldas @surajnarwade Thoughts?